### ISSUE TITLE
EC-Earth-Consortium ceda-cc report

### ISSUE THREAD

**Issue by mollymacrae** [cedadev/RAMIP#4]
The following error was raised from sampling the experiment ssp370 r5i1p1f1 2015-2020 data for EC-Earth-Consortium. 

**Incorrect activity_id**
For all files, the Global attribute activity_id is ''ScenarioMIP AerChemMIP' when it should be 'RAMIP' according to the tables.
FAILED:: Global attributes do not match constraints: [('activity_id', 'ScenarioMIP AerChemMIP', "['RAMIP']

**Comment by Declan-FMI** [comment]
I disagree, activity_id is specified as "ScenarioMIP AerChemMIP" for ssp370 experiments. "RAMIP" for all others. See RAMIP_CV.json, lines 11007-11009 in (I think) the latest version.

// Declan O'Donnell (declan.odonnell -at - fmi.fi)

**Comment by mollymacrae** [comment]
Ahh yes, I see this thank you!
I think our checks were picking this up as an error as the activity_id listed at the top ([line 44](https://github.com/cedadev/RAMIP/blob/88ce24bbd67a1b23c3d3eba260304e8c485b5925/RAMIP_CV.json#L44)) is set to "RAMIP"
Maybe it would be worth updating ssp370 in the RAMIP tables to:
 "activity_id": ["ScenarioMIP AerChemMIP RAMIP"]

**Comment by mollymacrae** [comment]
**Variable ccn02** standard_name attribute needs to be corrected before archival:
FAILED:: Variable [ccn02] has incorrect attributes: standard_name="cloud_condensation_nuclei_concentration_at_0point2_percent_supersaturation" [correct: "cloud_condensation_nuclei_concentration_at_0.2_percent_supersaturation"]

**Comment by Declan-FMI** [comment]
Hi,
No, it is correct as is. The standard name was changed  earlier this year, May or June if I remember correctly. 0.2 is the old, 0point2 the new. Laura can probable tell you why the change was necessary.
See
<https://github.com/ramipaerosols/RAMIP/blob/RAMIP-cmor-tables/RAMIP_RAMIPmon.json>
Best regards,
Declan
________________________________

**Comment by mollymacrae** [comment]
Thank you for confirming this.

Just to note, perhaps we could consider changing the standard_name to '0p2' for consistency with other CF standard names containing decimals
e.g. volume_absorption_coefficient_of_radiative_flux_in_air_due_to_**pm2p5**_dry_aerosol_particles
https://cfconventions.org/Data/cf-standard-names/current/build/cf-standard-name-table.html

**Comment by mollymacrae** [comment]
EC-Earth QC checks to do list:

- [ ] Update activity_id to "activity_id": ["ScenarioMIP AerChemMIP RAMIP"]
- [ ] change ccn02 standard_name to 'volume_absorption_coefficient_of_radiative_flux_in_air_due_to_pm2p5_dry_aerosol_particles' to match other CF standard_name formats handling a '.'
